src/build-order-actions.js

```diff
@@ -83,6 +92,55 @@ const buildOrderMethods = {
 
     return actions
   }, []),
+
+  returnItemState: (item, existingItem) => {
+    const actions = []
+    if (!existingItem)
+      return actions
+
+    if (item.shipmentState !== existingItem.shipmentState)
+      actions.push({
+        action: 'setReturnShipmentState',
+        returnItemId: item.id,
+        shipmentState: item.shipmentState,
+      })
+
+    if (item.paymentState !== existingItem.paymentState)
+      actions.push({
+        action: 'setReturnPaymentState',
+        returnItemId: item.id,
+        paymentState: item.paymentState,
+      })
+    return actions
+  },
+
+  returnInfo: (order, existingOrder) => {
```

Comment:
I just looked at this now, noticed that the update action was written here, I was expecting the SDK to be used in this case, returninfo sync action is present in the SDK

maybe we take a look